nanomq/apps/broker.c

```diff
@@ -227,15 +227,8 @@ server_cb(void *arg)
 				// avoid being free
 				conn_param_clone(work->cparam);
 			}
-			// restore clean session
-			char *clientid =
-			    (char *) conn_param_get_clientid(work->cparam);
-			if (clientid != NULL) {
-				restore_session(clientid, work->cparam,
```

Comment:
Why delete `restore_session`?

Reply:
Clean session functions would be moved to the protocol layer.